spec/models/contacts_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe Contact, type: :model do
+  fixtures :contacts
+  fixtures :users
+
+  context 'name is not empty' do
+    it 'return a count of 3 after saving a new entry in the contact table' do
+      Contact.create! name: "Contact3", user_id: 1
+      expect(Contact.all.count).to eq(3)
+    end
+  end
+
+  context 'name is empty' do
+    it 'return a validation error' do
+      expect { 
+        Contact.create! birthday:"2022-02-02", user_id: 1 
```

Comment:
As I mentioned before, since you're using fixtures you should use the identifiers defined in the fixture file rather than using ids directly:
```rb
# bad
Contact.create! birthday:"2022-02-02", user_id: 1
# ok
Contact.create! birthday:"2022-02-02", user: users(:one) # this refers to the fixture `one` in the fixture file.
# best
Contact.create! birthday:"2022-02-02", user: users(:dexter_morgan) # use descriptive names.
```